src/percy-agent-client/dom.ts

```diff
@@ -147,7 +147,7 @@ class DOM {
           break
         case 'textarea':
           // setting text or value does not work but innerText does
```

Comment:
We should update this comment too

Reply:
Heh, I realized that and then amended the commit